src/lib/apis/gemini.js

```diff
@@ -1,5 +1,8 @@
 // @ts-check
 
+import urljoin from "url-join"
 import fetch from "./fetch"
-const { GEMINI_API_BASE } = process.env
-export default (path, params) => fetch(`${GEMINI_API_BASE}/${path}`, params)
+import config from "config"
```

Comment:
#minor but u could do `import { GEMINI_API_BASE } from "config"` instead of importing the whole thing and assigning below. Notice you generally do it this way, the shortcut might save a line or two. But it doesn't matter!